Insights/EventsController.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  EventsController.swift
```

Comment:
If we rename the protocol from `EventProcessor` to `EventProcessable`, then I would name this class `EventProcessor `